source/_integrations/yolink.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: YoLink
+description: Instructions on how to integrate YoLink Devices into Home Assistant.
+ha_category:
+  - yolink
```

Comment:
Please only use the existing categories. See other integration pages for examples.